app/services/employee_importer.rb

```diff
@@ -0,0 +1,74 @@
+require "slack-ruby-client"
+
+class EmployeeImporter
+  def initialize
+    configure_slack
+  end
+
+  def import(dry_run = false)
+    import_results = {created: 0, skipped: 0, dry_run: false}
+    slack_user_finder = SlackUserFinder.new("", client)
+    total_employees = slack_user_finder.users_list.count
+
+    slack_user_finder.users_list.each_with_index do |user, index|
+      if !dry_run
+        success = import_employee(
+          slack_username: user["name"],
+          slack_user_id: user["id"]
+        )
+      else
+        success = dry_import_employee(
+          slack_username: user["name"],
+          slack_user_id: user["id"]
+        )
+        import_results[:dry_run] = true
+      end
+
+      if success
+        Rails.logger.info "#{index + 1}/#{total_employees}: Created #{user["name"]}#{" (dry run)" if dry_run}".green
```

Comment:
these statements might be more readable if wrapped into well-named private methods - eg `log_success` or something